### ISSUE TITLE
Rename trace types

### ISSUE THREAD

**Issue by coot** [input-output-hk/ouroboros-network#3926]
All trace types should follow the following naming convention `Trace<NAME>`.
Furthermore, all traces configurable by `cardano-node` should use the same type
name as the corresponding
[option](https://github.com/input-output-hk/cardano-node/blob/master/cardano-node/src/Cardano/Tracing/Config.hs?plain=1#L66-L118=).

**Comment by jutaro** [comment]
Please be aware that with the new tracing system these options disappear. Maybe you want to defer this change to a point after new-tracing has become the new standard?

**Comment by coot** [comment]
@jutaro the new tracing system is probably also using some kind of filtering of messages by tracer name, isn't it?  If so is this derived from trace type name?

**Comment by jutaro** [comment]
Yes, it uses hierarchical names as you can see here: https://github.com/input-output-hk/cardano-node/blob/master/doc/new-tracing/tracers_doc_generated.md
